Sources/PrimerSDK/Classes/Core/Checkout Components/PrimerHeadlessUniversalCheckoutUIManager.swift

```diff
@@ -109,14 +109,21 @@ extension PrimerHeadlessUniversalCheckout {
             log(logLevel: .debug, message: "🧨 deinit: \(self) \(Unmanaged.passUnretained(self).toOpaque())")
         }
         
+        public init(paymentMethodType: String) {
+            self.paymentMethodType = paymentMethodType
```

Comment:
This requires a check, otherwise what's going to happen if it gets initialized with an unsupported type?

Reply:
✅